src/index.md

```diff
@@ -245,7 +245,7 @@ const tidy = clone(SparkRetrievalResultCodes).flatMap(({ day, rates }) => {
       ${Plot.plot({
       title: 'Time to First Byte (ms)',
       x: { type: 'utc', ticks: 'day' }, // TODO: Change tick to month once we have more data
-      y: { grid: true },
+      y: { grid: true, zero: true},
```

Comment:
Interesting. I see that you are already using the `zero: true` option. Why do we need the fake datapoint then? Does this option not work as expected?

Reply:
I thought you wanted beginning value to be zero 🤦🏻 I get what you meant by that -- you wanted us to make label on Y axis to start from zero. It should be fixed now.